ci/scripts/cron-e2e-test.sh

```diff
@@ -4,7 +4,7 @@
 set -euo pipefail
 
 source ci/scripts/common.sh
-export RUN_COMPACTION=1;
```

Comment:
Why disable this?

Reply:
I found that a bug was introduced in #14259 when I ran the main-cron job. After discussion with @zwang28 , we decided to disable it temporarily and Zheng will open another PR to fix it.